src/main/scala/net/jcazevedo/moultingyaml/YamlValue.scala

```diff
@@ -17,12 +17,13 @@ sealed abstract class YamlValue {
 
   def print(flowStyle: FlowStyle = FlowStyle.DEFAULT,
             scalarStyle: ScalarStyle = ScalarStyle.DEFAULT,
-            lineBreak: LineBreak = LineBreak.DEFAULT) = {
-    val printer = new SnakeYamlPrinter(flowStyle, scalarStyle, lineBreak)
-    printer(this)
+            lineBreak: LineBreak = LineBreak.DEFAULT) : String = {
```

Comment:
I'm inclined to drop this method and leave just the one that accepts a `YamlPrinter`, with default value of `SnakeYamlPrinter(FlowStyle.DEFAULT, ScalarStyle.DEFAULT, LineBreak.DEFAULT)`. What do you think?